changelogs/fragments/7051-ipa-config-new-choice-idp-to-ipauserauthtype.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+ - ipa_config - adds ``idp`` choice to ``ipauserauthtype`` parameter's choices (https://github.com/ansible-collections/community.general/pull/7051).
+ 
```

Comment:
I dont know, how to get that redmark of there?

Reply:
The last line contains a single space, and afterwards there is no empty line (which the red mark indicates). Simply remove that space, and everything's good. (As long as the sanity tests don't complain, you can also leave that space there and ignore this mark.)

Reply:
Okay it was because I use VSCode and it thought I would like same indent :smile: I fixed it by force undoing commit and force pushing.